### PR TITLE
Add first end-to-end test

### PR DESCRIPTION
**Pull request by wlsf82**
Hi @miltonbsn and @idmarjr,

As promised, here's my first contribution to the project. 🚀 

Take a look and let me know what do you think?

In another PR, I intend to create a pipeline to run such tests.

### CODE REVIEW

**Comment by miltonbsn** [comment]
@wlsf82 amazing contribution! Certainly raising our level! ⭐  ⭐ ⭐ ⭐ ⭐ 

Just a point to discussion... 

As we also have some unit test made with jest, could you recommend a good practice to manage the execution through yarn (unit, e2e, all...) ?

**Comment by idmarjr** [comment]
I manage to run  the test after some digging.
Maybe it worth to add to `README.md` some instructions about the need to create `cypress.env.example.json` with the username and password before try to run the tests.

OR, we can create a default test user and let it live there, removing `cypress.env.json` from `.gitignore`.  
Not sure what's the best practice here. :)

**Comment by wlsf82** [comment]
> I manage to run the test after some digging. Maybe it worth to add to `README.md` some instructions about the need to create `cypress.env.example.json` with the username and password before try to run the tests.

I added such info in the README. See line 48.

File: README.md [hunk 1]
```diff
@@ -37,6 +37,16 @@ If you already have docker installed you can skip the [Setup](#setup) and [Run](
...
 
+### Tests
+
+With the database, backend, and frontend up-and-running:
+
+Run `yarn test` to run Cypress tests in headless mode.
+
+Or, run `yarn cy:open` to open the test runner for running tests in interactive mode.
+
+> **Note:** For the tests to work, you will have to copy the `cypress.env.example.json` file as `cypress.env.json` (included on `.gitignore`) and update the `user` and `password` with your credentials.
```
Comment (idmarjr) [inline README.md]:
```suggestion
> **Note:** For the tests to work, you will have to copy the `cypress.env.example.json` file as `cypress.env.json` (included on `.gitignore`) and update the `user` and `password` with valid credentials.
```

**Review by idmarjr — APPROVED** [review]
Everything looks good :)
It's awesome to see tests becoming a reality in the project.

Thanks a ton for the contributions @wlsf82

**Comment by idmarjr** [comment]
@all-contributors please add @wlsf82 for test

**Pull request merged**